lib/loading.dart

```diff
@@ -47,10 +47,8 @@ class LoadingOverlay {
 
   OverlayEntry _createdLoadingEntry(BuildContext context) {
     return OverlayEntry(
-      builder: (BuildContext context) => Stack(
-        children: <Widget>[
-          if (loadingUi != null) loadingUi! else Loading(),
-        ],
+      builder: (BuildContext context) => AbsorbPointer(
+        child: (loadingUi != null) ? loadingUi! : Loading(),
```

Comment:
I think it'd be better if we were to include an option to whether block all pointers or not instead.